test/skimNtuple_HHbtag.cpp

```diff
@@ -611,7 +611,8 @@ int main (int argc, char** argv)
   int smearVariation = (isMC ? gConfigParser->readIntOption("JetSmearing::smearVariation") : 0);
   cout << "** INFO: doSmearing? " << doSmearing << endl;
   cout << "** INFO: smearVariation: " << smearVariation << " [0:nominal  +1:up  -1:down]" << endl;
-  SmearedJetProducer Smearer(wpyear, doSmearing, smearVariation, isPostVFP);
+  SmearedJetProducer Smearer_AK4(puYear, doSmearing, smearVariation, false);
+  SmearedJetProducer Smearer_AK8(puYear, doSmearing, smearVariation, true);
```

Comment:
this assumes that it's fine to always apply the same variations to AK4 & AK8; I would tend to think it is OK, but probably something we'd want to make sure of

Reply:
Those two objects end up being independent, as they are considered for different categories (resolved and boosted). Therefore, we would have at most to combine AK4 up variations with AK8 down variations only when fitting histograms, which is done later on.